CONTRIBUTING.md

```diff
@@ -40,8 +40,10 @@ Here are a few things you can do that will increase the likelihood of your pull
 
 ## Development setup
 
-pip instal -r requirements.txt
-pip instal -r requirements-dev.txt
+```
+$ pip install -r requirements.txt
+$ pip install -r requirements-dev.txt
```

Comment:
The code/pre block is a good idea here, but if you don't mind remove the $ so that they can be copy-n-pasted without editing. Otherwise LGTM.

Reply:
Makes sense, updated.